src/Processors/Formats/IRowInputFormat.cpp

```diff
@@ -148,6 +142,8 @@ Chunk IRowInputFormat::generate()
                     }
                 }
 
+                ++total_rows;
```

Comment:
Moved afrer calling `readRow` so it will be 0 in the first `readRow` call, it's more convinient than 1